rviz_default_plugins/src/rviz_default_plugins/tools/pose_estimate/initial_pose_tool.cpp

```diff
@@ -57,6 +58,16 @@ InitialPoseTool::InitialPoseTool()
 
   qos_profile_property_ = new rviz_common::properties::QosProfileProperty(
     topic_property_, qos_profile_);
+
+  covariance_x_property_ = new rviz_common::properties::FloatProperty(
+    "Covariance x", 0.5 * 0.5, "Covariance on the x-axis.",
+    getPropertyContainer(), 0, this);
+  covariance_y_property_ = new rviz_common::properties::FloatProperty(
+    "Covariance y", 0.5 * 0.5, "Covariance on the y-axis.",
+    getPropertyContainer(), 0, this);
+  covariance_yaw_property_ = new rviz_common::properties::FloatProperty(
+    "Covariance yaw", M_PI / 12.0 * M_PI / 12.0,
```

Comment:
I believe the Windows compiler is complaining about this line (truncation from `double` to `float`). I suggest wrapping it in `static_cast`:
```suggestion
    "Covariance yaw", static_cast<float>(M_PI / 12.0 * M_PI / 12.0),
```